docs/app/Examples/collections/Message/Variations/MessageInfoPropsExample.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import { Message } from 'stardust'
+
+const MessageInfoPropsExample = () => (
+  <Message
+    info
+    header='Was this what you wanted?'
+    content="Did you know it's been a while?"
+  />
+)
+
+export default MessageInfoPropsExample
```

Comment:
Valid point.  Without the code snippet showing, it is not clear what the difference is.  I'll add some description text.